acceptance/analyzer_test.go

```diff
@@ -835,47 +992,50 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 					otherLayersDir, ctrPath("/other-layers"),
 					analyzeImage,
 					h.WithFlags(
-						"--env", "CNB_PLATFORM_API=0.4",
+						"--env", "CNB_PLATFORM_API="+apiString,
 					),
 					h.WithArgs(
 						ctrPath(analyzerPath),
 						"-layers", ctrPath("/other-layers"),
-						"-cache-dir", ctrPath("/cache"), // use a cache so that we can observe the effect of group.toml on /some-other-layers (since we don't have a previous image)
+						"-cache-dir", ctrPath("/cache"), // use a cache so that we can observe the effect of group.toml (since we don't have a previous image)
 						"some-image",
 					),
 				)
-				h.AssertPathExists(t, filepath.Join(otherLayersDir, "some-buildpack-id"))
+				h.AssertPathExists(t, filepath.Join(otherLayersDir, "some-buildpack-id")) // some-buildpack-id is found in the working directory: /layers/group.toml
 
-				h.DockerCopyOut(t, containerName, ctrPath("/layers"), layersDir)
+				h.DockerCopyOut(t, containerName, ctrPath("/layers"), layersDir) // analyzed.toml is written at the working directory: /layers
 				assertAnalyzedMetadata(t, filepath.Join(layersDir, "analyzed.toml"))
 			})
 		})
-	})
 
-	when("Platform API = 0.5", func() {
 		when("layers path is provided", func() {
 			it("uses the group path at the layers path and writes analyzed.toml at the layers path", func() {
```

Comment:
can we put this `it` under the `when` at line 978?

Reply:
6d7c623